readme.md

```diff
@@ -1 +1,12 @@
-# Dev Ops
\ No newline at end of file
+# Dev Ops
+
+## Learning Objectives
+
+- Explain what continuous integration is and what problem it solves
+- Set up a github repository that runs Travis builds
```

Comment:
Wow, will you really have time to cover this? I've never used Travis so don't know!